languages.js

```diff
@@ -34,20 +34,6 @@ module.exports = {
       return /^[1-9]{1}(,[0-9]+)?E{1}$/.test(token);
     }
   },
-  'de-DE-COM': {
-    true: 'WAHR',
-    false: 'FALSCH',
-    verticalSeparator: ';',
-    horizontalSeparator: ',',
-    argumentSeparator: ',',
-    decimalSeparator: '.',
-    reformatNumberForJsParsing: function (n) {
-      return n;
-    },
-    isScientificNotation: function (token) {
-      return /^[1-9]{1}(\.[0-9]+)?E{1}$/.test(token);
-    }
-  },
```

Comment:
Should we remove this entire languages file or keep it for now incase we have to undo our language helper logic?
Inside index.js we only use languages to display the valid language options for the tokenizer.